contrib/rosetta/docker-compose.yaml

```diff
@@ -29,7 +29,7 @@ services:
     working_dir: /rosetta
     command: ["python3", "faucet.py"]
     expose:
-      - 8080
+      - 8000
```

Comment:
are you sure about this change?

Reply:
faucet.py listens on [8000](https://github.com/cosmos/cosmos-sdk/blob/8133ee8048a114679743b730fd796f3a8d5cb59a/contrib/rosetta/configuration/faucet.py#L24)

but i think it should be separate pr